projects/js-packages/partner-coupon/components/redeem-partner-coupon-post-connection/style.scss

```diff
@@ -0,0 +1,184 @@
+@import '@wordpress/base-styles/breakpoints';
+@import '@wordpress/base-styles/mixins';
+@import '@automattic/jetpack-base-styles/style';
+
+.jetpack-redeem-partner-coupon-post-connection {
+	background: var( --jp-white );
+	box-shadow: 0 0 40px rgba(0, 0, 0, 0.08);
+	border-radius: 4px;
+
+	/*
+	 * Make our custom "Redeem" button look like the "Setup and redeem" button.
+	 * We have to do this because it's not native to the ConnectScreen component.
+	 */
+	.components-button {
+		font-weight: 600;
+		font-size: 14px;
+		border-radius: 4px;
+		height: 40px;
+		justify-content: center;
+	}
+
+	&__layout {
+		display: grid;
```

Comment:
If we wait with defining `display: grid` until `@break-mobile` or with the rest of the grid values in `@break-medium`, the mobile behaviour stays "in line" for quite a bit longer (I think the button starts going outside the box at ~330px).

With `display: grid` @ 330px width:
![Screenshot 2022-02-09 at 14 59 40](https://user-images.githubusercontent.com/3846700/153216438-87520c6d-7d6d-4939-bb45-84552d4d7f9a.png)

Without `display: grid` @ 330px width:
![Screenshot 2022-02-09 at 14 59 47](https://user-images.githubusercontent.com/3846700/153216487-fa352955-c00b-4b88-b7f3-d180723af843.png)